src/Accounts/Accounts/help/Get-AzProfile.md

```diff
@@ -0,0 +1,107 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Accounts.dll-Help.xml
+Module Name: Az.Accounts
+online version: https://docs.microsoft.com/en-us/powershell/module/az.accounts/get-azprofile
+schema: 2.0.0
+---
+
+# Get-AzProfile
+
+## SYNOPSIS
+Get the service profiles supported by installed modules.
+
+## SYNTAX
+
+```
+Get-AzProfile [-ModuleName <String[]>] [-ListAvailable] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Get the service profiles supported by installed modules.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-AzProfile -Module Az.Compute
```

Comment:
@markcowl please add the corresponding output for this example (you'll also want to regenerate the help when removing the `SupportsShouldProcess`)